keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -142,10 +143,14 @@ class WordPieceTokenizer(tokenizer.Tokenizer):
     be either an integer or string type.
 
     Args:
-        vocabulary: A list of strings or a string filename path. If
-            passing a list, each element of the list should be a single
-            WordPiece token string. If passing a filename, the file should be a
-            plain text file containing a single WordPiece token per line.
+        vocabulary: A list of strings, a string filename path, or
```

Comment:
Should we use this `vocabulary` arg to load the pretrained vocab, or make a new arg `from_pretrained` (`from_pretrained='bert_uncased'`)? I am leaning towards the later one, since it explicitly conveys we are using pretrained tokenizers. @mattdangerw Matt, thoughts on this?

Reply:
Hmm, I think if we want to go with something like `from_pretrained` we need to think of a good uniform UX we can across the library. That is why it works for huggingface.

I was imaging we only offer a single parameter `lang="es"` or `lang="en"` and combine that with the `vocabulary_size` parameter.

I don't really think we should answer, how to get a BERT tokenizer, in the word piece tokenizer API, that's a bad separation of concerns. If you want the BERT tokenizer, you can get it through the BERT modeling API itself.

Reply:
added lang and vocab size parameters